core/src/main/java/hudson/model/queue/AbstractQueueSorterImpl.java

```diff
@@ -35,17 +35,13 @@ public int compare(BuildableItem lhs, BuildableItem rhs) {
      * sign(a-b).
      */
     protected static int compare(long a, long b) {
-        if (a>b)    return 1;
-        if (a<b)    return -1;
-        return 0;
+        return Long.compare(a, b);
     }
 
     /**
      * sign(a-b).
      */
     protected static int compare(int a, int b) {
```

Comment:
Similarly here with `Integer.compare`

Reply:
I addressed your comments and marked the methods as deprecated and suggested Integer.compare